external/netfx/Configurations.props

```diff
@@ -3,7 +3,12 @@
   <PropertyGroup>
     <!-- Include netcoreapp since we need to generate facades between desktop and netcoreapp -->
     <BuildConfigurations>
+      net45;
+      net451;
```

Comment:
We did before.  Some facades were being built against this for the API versions that target netstandard1.1 or lower.